dts/arm/nuvoton/npcx/npcx7-alts-map.dtsi

```diff
@@ -127,9 +128,9 @@
 
 		/* SCFG DEVALT D */
 		altd_psl_in1_ahi:  altd0     { alts = <&scfg 0x0D 0x0 0>; };
-		altd_npsl_in1_sl:  altd1     { alts = <&scfg 0x0D 0x1 0>; };
+		altd_npsl_in1_sl:  altd1     { alts = <&scfg 0x0D 0x1 1>; };
```

Comment:
Indeed. Thanks.

Reply:
Done.

Reply:
Also, how would you override this field back to 0 if you needed to in a sub-board?

It might be worth making a comment in the default pinctrl section of the device tree file of how to override the default values.

Reply:
I will add more comments in soc_pins.h

Reply:
Done.